### PR TITLE
FE-1223 Dependencies Updates

### PR DESCRIPTION
**Pull request by gfriedmansp**
### What Changed
 - Upgraded dependency
   - papaparse
 - Upgrade dev-dependencies
   - jest
   - node-sass
   - sass-loader
   - wait-on
   - webpack
   - webpack-bundle-analyzer
   - webpack-dev-server

### How To Test
 - Smoke test: https://docs.google.com/spreadsheets/d/1pw9vlNnGsNG6sRruhuMthgnHti_812vknCwoivWrQ4c/edit#gid=0

### To Do
- [ ] Fix any bugs that come up

### CODE REVIEW

File: config/jest/setup.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ beforeEach(() => {
 // Mock moment to set a default timezone
 jest.mock('moment', () => {
   jest.unmock('moment'); // must unmock, so moment-timezone can require
-  const momentTimezone = require.requireActual('moment-timezone');
+  const momentTimezone = jest.requireActual('moment-timezone');
```
Comment (gfriedmansp) [inline config/jest/setup.js:36]: They deprecated `require.requireActual` in jest@^25 and completely removed it in jest@^26. We bumped from 24 to 26, so 🤷

File: config/webpack.config.js [hunk 2]
```diff
@@ -574,28 +574,19 @@ module.exports = function(webpackEnv) {
         }),
       isEnvProduction &&
         new BundleAnalyzerPlugin({
-          // Can be `server`, `static` or `disabled`.
-          // In `server` mode analyzer will start HTTP server to show bundle report.
-          // In `static` mode single HTML file with bundle report will be generated.
-          // In `disabled` mode you can use this plugin to just generate Webpack Stats JSON file by setting `generateStatsFile` to `true`.
+          // See Docs for these options: https://www.npmjs.com/package/webpack-bundle-analyzer
```
Comment (gfriedmansp) [inline config/webpack.config.js:577]: I just got rid of a bunch of documentation here that was out of date and/or unnecessary. Link to the docs > copying docs into our codebase

File: package.json [hunk 3]
```diff
@@ -91,7 +91,7 @@
     "localforage": "^1.5.3",
     "lodash": "^4.17.19",
     "moment": "^2.20.1",
-    "papaparse": "^4.3.6",
+    "papaparse": "^5.3.0",
```
Comment (gfriedmansp) [inline package.json:94]: The only non-dev-dependency upgrade

File: src/components/datePicker/tests/__snapshots__/DatePicker.test.js.snap [hunk 4]
```diff
@@ -8,7 +8,7 @@ exports[`Component: DatePicker handleFormDates should use a date field format if
   >
     Narrow Date Range
   </label>
-  <ForwardRef
+  <TextField
```
Comment (gfriedmansp) [inline src/components/datePicker/tests/__snapshots__/DatePicker.test.js.snap:11]: Jest 25.1.0 added 
`[pretty-format] Fix pretty-format to respect displayName on forwardRef (#9422)`

**Review by nstanard — APPROVED** [review]
LGTM!

**Pull request merged**